framework/sessions/__init__.py

```diff
@@ -166,9 +165,13 @@ def before_request():
             session = Session.load(session_id) or Session(_id=session_id)
         except itsdangerous.BadData:
             return
-        if session.data.get('auth_user_id') and 'api' not in request.url:
-            database['user'].update({'_id': session.data.get('auth_user_id')}, {'$set': {'date_last_login': datetime.utcnow()}}, w=0)
-        set_session(session)
+
+        delta = (datetime.utcnow() - session.date_created).total_seconds()
```

Comment:
Yup. Saman didn't either before she implemented it recently, which is how I found out about it. :)